ui/src/app/workflows/components/retry-workflow-node-panel.tsx

```diff
@@ -0,0 +1,100 @@
+import {Checkbox} from 'argo-ui/src/components/checkbox';
+import {Tooltip} from 'argo-ui/src/components/tooltip/tooltip';
+import React, {useState} from 'react';
+
+import {Parameter, RetryOpts, Workflow} from '../../../models';
+import {ParametersInput} from '../../shared/components/parameters-input';
+import {services} from '../../shared/services';
+import {Utils} from '../../shared/utils';
+import {ErrorNotice} from '../../shared/components/error-notice';
+
+interface Props {
+    nodeId: string;
+    workflow: Workflow;
+    isArchived: boolean;
+    isWorkflowInCluster: boolean;
+    onRetrySuccess: () => void;
+}
+
+export function RetryWorkflowNode(props: Props) {
+    const [overrideParameters, setOverrideParameters] = useState(false);
+    const [restartSuccessful, setRestartSuccessful] = useState(false);
+    const [workflowParameters, setWorkflowParameters] = useState<Parameter[]>(JSON.parse(JSON.stringify(props.workflow.spec.arguments.parameters || [])));
+    const [error, setError] = useState<Error>();
+
+    async function submit() {
+        const parameters: RetryOpts['parameters'] = overrideParameters
+            ? [...workflowParameters.filter(p => Utils.getValueFromParameter(p) !== undefined).map(p => p.name + '=' + Utils.getValueFromParameter(p))]
+            : [];
+        const opts: RetryOpts = {
+            parameters,
+            restartSuccessful,
+            nodeFieldSelector: `displayName=${props.nodeId}`
```

Comment:
Hello, shouldn't this be `id=${props.nodeId}` since you are using `nodeId` and not `displayName`?

Reply:
@djanjic I couldn't find `id` as a possible options in the doc [here](https://argo-workflows.readthedocs.io/en/latest/node-field-selector/#possible-options), but it seems to be working

Reply:
I guess that doc is not updated to address this change. I tracked it down, `id` as selector is introduced here: https://github.com/argoproj/argo-workflows/pull/6885

Reply:
For back-link reference, this was documented in https://github.com/argoproj/argo-workflows/pull/13463